phpunit.xml.dist

```diff
@@ -31,4 +28,12 @@
             </exclude>
         </whitelist>
     </filter>
+    <groups>
+        <exclude>
+            <group>benchmark</group>
```

Comment:
why adding that ?

Reply:
There is a single test in the `benchmark` group, but it's currently not working (no data generated in the test + it needs a real mysql setup). So I've kept it out for now, this can be a separate github action workflow imo